kernelci/runtime/__init__.py

```diff
@@ -134,6 +134,7 @@ def get_params(self, job, api_config=None):
             'runtime_image': job.config.image,
         }
         params.update(job.config.params)
+        params.update(job.platform_config.params)
```

Comment:
Right, and then we can remove the other `platform_config` then?

Reply:
Note: I might be missing some context - please correct me if I'm wrong.

L137 adds to the template rendering `params` only the platform (`device_types`) `params` section, e.g.

```
device_types:
  minnowboard-turbot-E3826:
    arch: x86_64
    boot_method: grub
    mach: x86
    params:
      <added>: <param>
```

Required fields of the platform configuration (like `boot_method`, kernelci/legacy/config/test.py:L23) would still be unavailable unless duplicated in the `params` section.

I thought `platform_config.params` were meant to override any previously set value for a job (e.g. in a `job.config`).

I'll verify these assumptions and either post container image used or delete this comment in order not to cause any further confusion.

Reply:
Ah yes sorry, it's a different set of parameters.  I think I got mislead because the previous PR added the other line while removing this one.  So this is fine.